README.rst

```diff
@@ -47,6 +47,8 @@ Requirements
 * Python 3.5+
 * Django 2.0+
 * tablib 0.14+
+* openpyxl >=2.4,<2.5
+* diff-match-patch
```

Comment:
Hey @SteveHere , thanks for the contrib! How about instead adding these to to the readme, we remove tablib? From a user point of view, 99% of the time you shouldnt care that we have these base dependencies. Only that our project is built against Django 2.0+ and Python 3.5+.

Reply:
Sorry for the long pause; I don't really browse this site as often as I should.

As for removing tablib, it would be nicer to just rip that dependency out. Should take a while to search through it all.

Reply:
Just looked at where tablib is used... it is extensive for BASE_FORMATS. Rewriting this to use something else will be a nightmare. Best to just remove the other 2 dependencies instead (openpyxl & diff-match-patch).

Reply:
OK then Im going to close this. Feel free to open another to fix up the readme section!